readthedocs/proxito/views/serve.py

```diff
@@ -164,10 +164,12 @@ def get(self, request, proxito_path, template_name='404.html'):
         the Docs default page (Maze Found) is rendered by Django and served.
         """
         # pylint: disable=too-many-locals
+        log.info('Executing handler404. proxito_path=%s', proxito_path)
```

Comment:
This isn't configured as a handler404. We should just say `404 handler`, since that has a specific name that isn't correct in this case.

Reply:
He... it's configured as a `handler404` when running tests. But yeah, in production it's different. I'll update the string.